sdk/go/common/workspace/templates.go

```diff
@@ -502,13 +502,25 @@ func CopyTemplateFilesDryRun(sourceDir, destDir, projectName string) error {
 func CopyTemplateFiles(
 	sourceDir, destDir string, force bool, projectName string, projectDescription string,
 ) error {
+	// Needed for stringifying numeric user-provided project names.
+	yamlName := projectName
```

Comment:
Why are we declaring this (and description below) to one value but unconditionally overwriting that with the YAML value. Looks like we can just drop these first declarations.

Reply:
I'm guessing Kyle was trying to convert the `[]byte` to string early.

@dixler we can do either:

```go
yamlName, err := yaml.Marshal(projectName) 
// ...
name = string(yamlName)
```

or if you'd prefer for yamlName to be a string, a helper function `yamlName, err := yamlToString(projectName)`.

Reply:
was copying the existing code. can clean it up a bit more.

Reply:
We will error if the project name or description was unable to be serialized to YAML for some reason rather than only if we are serializing to YAML.